README.md

```diff
@@ -7,28 +7,26 @@
 
 # Working Features
 
-This MVP Python client enables the execution of **[WebAssembly](https://en.wikipedia.org/wiki/WebAssembly) Polywrappers** *(or just "wrappers")* on a python environment. It's built following the functionality of the [JavaScript Polywrap Client](https://github.com/polywrap/toolchain), which is currently more robust out and battle tested, as it has additional capabilities than this MVP.
+This MVP Python client enables the execution of **[WebAssembly](https://en.wikipedia.org/wiki/WebAssembly) Polywrappers** *(or just "wrappers")* on a python environment, regardless of what language this wrapper was built in.
 
-In the future, the Polywrap DAO will continue improving this Python capabilities to reach feature parity with the JS stack, including the possibiity of creating WASM wrappers with Python code. 
+The client is built following the functionality of the [JavaScript Polywrap Client](https://github.com/polywrap/toolchain), which is currently more robust and battle tested, as it has additional capabilities than this MVP. In the future, the Polywrap DAO will continue improving this Python capabilities to reach feature parity with the JS stack.
 
 Here you can see which features have been implemented on each language, and make the decision of which one to use for your project.
 
 | Feature | [Python](https://github.com/polywrap/python-client) | [Javascript](https://github.com/polywrap/toolchain) | 
 | -- | -- | -- |
 | Invoke wrappers | yes | yes |
-| polywrap-asyncify | replaced with wasmtime | yes |
-| polywrap-uri-resolvers | legacy | refactored |
-| polywrap-msgpack| yes, tested 100% | yes |
-| Subinvoke | wip | yes |
-| Wrap Manifest | WIP | yes |
-| Client Config Builder Package | wip | yes |
+| Subinvoke wrappers | wip | yes |
```

Comment:
not wip but fully implemented, I'd suggest using emoji here would be better

✅  for fully working, ⚠️ for partially working , ❌  or 🚫 for not working

Reply:
Let's be less verbose and call invoke wrappers -> invoke, env configuration -> env, etc

Reply:
Would it be worth having a column describing the feature though? I think having a brief description is useful to digest the components of the client!

Agreed on the emojis, updating now